concordium-node/src/configuration.rs

```diff
@@ -93,27 +93,27 @@ pub struct PrometheusConfig {
         default_value = "127.0.0.1",
         env = "CONCORDIUM_NODE_PROMETHEUS_LISTEN_ADDRESS"
     )]
-    pub prometheus_listen_addr:   String,
+    pub prometheus_listen_addr: String,
```

Comment:
I think we should change this to `IpAddr` https://doc.rust-lang.org/std/net/enum.IpAddr.html